src/questions/intro-to-defi.yaml

```diff
@@ -1,19 +1,354 @@
+  answerKeys:
+    - A
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: Unpaid Centralized loans lead to loss for banks
+      key: A
+    - content: Bank loans are easy to get
+      key: B
+    - content: Banks differentiate between potential borrowers based on the credit score
+      key: C
+    - content: Banks charge low-interest rates
+      key: D
+- uuid: 078e0e8c-4da2-4db9-87d9-7d70bced38cf
+  type: MultipleChoice
+  content: How can someone access the world of DeFi?
+  hint: NoHint
+  explanation: Decentralized Finance can be accessed using the internet, decentralised applications, and a blockchain address.
+  answerKeys:
+    - B
+    - C
+    - D
+  difficultyLevel: "Low"
+  choices:
+    - content: Using Bank accounts
+      key: A
+    - content: Using Decentralized Applications
+      key: B
+    - content: Internet access
+      key: C
+    - content: Blockchain address
+      key: D
+- uuid: 89ed030b-7eb1-4054-8627-4124bf72641a
+  type: MultipleChoice
+  content: Money transfer using banks is cumbersome because?
+  hint: Diadvantages of TradFi
+  explanation: In TradFi funds transfer can take upto 2 days and these transactions need to be significant, micro ones are not allowed. Transfer charges are quite high.
+  answerKeys:
+    - A
+    - B
+    - C
+  difficultyLevel: "Low"
+  choices:
+    - content: Fund transfer can take a long time
+      key: A
+    - content: Banks do not allow micro-transactions
+      key: B
+    - content: Bank levy charges on transactions
+      key: C
+    - content: None of these
+      key: D
+- uuid: 76ee573d-4190-405a-9712-f230cd2ec0e5
+  type: MultipleChoice
+  content: DeFi can replace which of the following traditional processes?
+  hint: Features of DeFi
+  explanation: DeFi can be used to conduct each of these listed processes efficiently and at cheaper rate compared to the traditional ways.
+  answerKeys:
+    - A
+    - B
+    - C
+  difficultyLevel: "Medium"
+  choices:
+    - content: Money Transfers
+      key: A
+    - content: Lending and Borrowing
+      key: B
+    - content: Currency exchange
+      key: C
+    - content: None of the above
+      key: D
+- uuid: 84196064-3dc8-4704-824a-a0d239a362c4
+  type: SingleChoice
+  content: What do Flash loans mean?
+  hint: Differenciating factor between normal loans and flash loans
+  explanation: Flash loans are instant loan using smart contracts which is granted to anyone without any collateral.
+  answerKeys:
+    - D
+  difficultyLevel: "High"
+  choices:
+    - content: Slow loan processing without collateral
+      key: A
+    - content: Instant loan processing with collateral
+      key: B
+    - content: Slow loan processing with collateral
+      key: C
+    - content: Instant loan processing without collateral
+      key: D
+- uuid: 43680d70-5e2d-4b55-9db4-ecc80e0246e2
+  type: SingleChoice
+  content: What makes DeFi technology so innovative?
```

Comment:
Question - What makes DeFi technology so innovative?
Answer - Blockchain allows developers to program money and move funds with a few lines of code. A single developer can write useful application in DeFi by writing a simple smart contract. Since TradFi is a closed ecosystem, it is a lot more work to do anything on top of existing Banks.